src/scss/06_components/paragraphs/_contact-card-listing.scss

```diff
@@ -0,0 +1,25 @@
+.contact-card-listing--container {
+  align-items: flex-start;
+  display: flex;
+  flex-direction: column;
+  flex-wrap: wrap;
+
+  @include breakpoint($breakpoint-s) {
+    flex-direction: row;
+  }
+
+  .contact-card {
+    margin-bottom: $spacing-triple;
+
+    @include breakpoint($breakpoint-s) {
+      min-width: 252px;
```

Comment:
Where does this [magic number](https://css-tricks.com/magic-numbers-in-css/) come from?